Makefile

```
@@ -15,7 +16,7 @@ BLUE=\033[0;34m
 CYAN=\033[0;36m
 NC=\033[0m # No Color
 
-.PHONY: help dev dev-ui build run install-air clean test install-ui work-init work-clean
+.PHONY: help dev dev-ui build run install-air clean test install-ui work-init work-clean docs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)